pulsar-client-api/src/main/java/org/apache/pulsar/client/api/Schema.java

```diff
@@ -213,6 +213,16 @@ default void configureSchemaInfo(String topic, String componentName,
         return DefaultImplementation.newProtobufSchema(SchemaDefinition.builder().withPojo(clazz).build());
     }
 
+    /**
+     * Disable SchemaInfo.
```

Comment:
The javadoc should illustrate what's the behavior for this option: 
 * We're just using the SerDe capabilities
 * No server side validation or enforcement of schema definition